src/main/java/de/tum/in/www1/artemis/service/AssessmentService.java

```diff
@@ -86,7 +95,7 @@ Result submitResult(Result result, Exercise exercise, Double calculatedScore) {
      * @return the updated Result
      */
     // NOTE: transactional makes sense here because we change multiple objects in the database and the changes might be invalid in case, one save operation fails
-    @Transactional
+    @Transactional // ok
```

Comment:
Seeing the comment above explaining why we use `@Transactional` here, does it make much sense to additionally add `// ok` (which doesn't explain much in itself)?